pkg/storageos/webhook.go

```diff
@@ -60,47 +65,60 @@ func (s Deployment) createWebhookService() error {
 // webhooks.  The configuration will be updated by the api-manager to include
 // the CA cert bundle.
 func (s Deployment) createMutatingWebhookConfiguration() error {
-	failurePolicy := admissionv1.Ignore
-	matchPolicy := admissionv1.Exact
-	sideEffects := admissionv1.SideEffectClassNoneOnDryRun
-	svcPort := webhookPort
 	scopeAll := admissionv1.AllScopes
-	path := "/mutate-pods"
-
 	webhooks := []admissionv1.MutatingWebhook{
-		{
-			Name: webhookPodMutatorName,
-			ClientConfig: admissionv1.WebhookClientConfig{
-				Service: &admissionv1.ServiceReference{
-					Name:      WebhookServiceName,
-					Namespace: s.stos.Spec.GetResourceNS(),
-					Port:      &svcPort,
-					Path:      &path,
+		s.mutatingWebhookConfiguration(webhookPodMutatorName, mutatePodPath, admissionv1.Ignore, []admissionv1.RuleWithOperations{
+			{
+				Operations: []admissionv1.OperationType{admissionv1.Create},
+				Rule: admissionv1.Rule{
+					APIGroups:   []string{""},
+					APIVersions: []string{"v1"},
+					Resources:   []string{"pods"},
+					Scope:       &scopeAll,
 				},
 			},
-			Rules: []admissionv1.RuleWithOperations{
-				{
-					Operations: []admissionv1.OperationType{admissionv1.Create},
-					Rule: admissionv1.Rule{
-						APIGroups:   []string{""},
-						APIVersions: []string{"v1"},
-						Resources:   []string{"pods"},
-						Scope:       &scopeAll,
-					},
+		}),
+		s.mutatingWebhookConfiguration(webhookPVCMutatorName, mutatePVCPath, admissionv1.Ignore, []admissionv1.RuleWithOperations{
```

Comment:
Didn't we decide to make the failure policy to fail for PVC mutating webhooks? https://github.com/storageos/api-manager/pull/42#discussion_r612606661

Reply:
Yes, I just wanted to enable it separately after doing more testing: https://github.com/storageos/cluster-operator/pull/318#discussion_r613880899